docs/testing.adoc

```diff
@@ -17,7 +17,9 @@ $ gulp test:unit
 ----
 
 == Functional Tests
+The planner functional tests can be executed against 2 backends
 
+=== 1. Mock Backend - Running Tests against `inmemory` (mock) data
```

Comment:
@jarifibrahim there are many minor things such as changing 2>two, full stops at end of sentences, Capital letter for product names like Selenium etc, basically following TW conventions .
But I do not think that's reason to stop this PR from going through as long as it is technically verified.